hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneShellHA.java

```diff
@@ -753,6 +754,33 @@ public void testLinkBucketOrphan() throws Exception {
     }
   }
 
+  @Test
+  public void testListBucket() throws Exception {
+    final String hostPrefix = OZONE_OFS_URI_SCHEME + "://" + omServiceId;
+    OzoneConfiguration clientConf =
+            getClientConfForOFS(hostPrefix, cluster.getConf());
+    int pageSize = 20;
+    clientConf.setInt(OZONE_FS_LISTING_PAGE_SIZE, pageSize);
```

Comment:
`FileSystem` may already be cached, so updated config may not be applied.

```suggestion
    clientConf.setInt(OZONE_FS_LISTING_PAGE_SIZE, pageSize);
    URI uri = FileSystem.getDefaultUri(clientConf);
    clientConf.setBoolean(String.format("fs.%s.impl.disable.cache", uri.getScheme()), true);
```

(needs `import java.net.URI`, too)